change_of_vars_demo1d/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    def latexify(
</pre><p>We should not add latexify code in notebook and also figure renders in notebook should not be latexified, Resaon is, as you can see this latexified figure dimensions are lesser than matplotlib default. So when user open notebook in github, he will not able to see it properly.</p><p> </p><p>Since you are doing in colab you are facing problem to run latexify command, right? I can see following solutions</p><ol><li>You can access terminal of google colab with your PC using SSH connection, you need to google how to do that, once you have access of colab's terminal,  then you can do run this command in fork of original repo.</li><li>You can install WSL in your windows PC.  This will enable ubuntu terminal in windows.</li><li>Use VM of ubuntu. (this may take load if you have less RAM)</li><li>Install Ubuntu with windows. (if I were you, I would do this)</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>